src/Factory.php

```diff
@@ -19,12 +19,13 @@ public static function create(
         LoopInterface $loop,
         array $options = []
     ): Client {
+        $container = self::createContainer($loop, $options);
         return new Client(
-            self::createContainer($loop, $options)
+            $container->get(CommandBusInterface::class)
         );
     }
 
-    private static function createContainer(LoopInterface $loop, array $options): ContainerInterface
+    public static function createContainer(LoopInterface $loop, array $options): ContainerInterface
     {
         $container = new ContainerBuilder();
```

Comment:
While you removed PHP-DI from dependencies this `ContainerBuilder` is still a concrete class from PHP-DI.